### ISSUE TITLE
Sunken Temple Avatar of Hakkar Boss Spawn for The God Hakkar quest has issues

### ISSUE THREAD

**Issue by tubtubs** [slowtorta/turtlewow-bug-tracker#1130]
**Issues*

1. Should your group wipe, fail, or reset the encounter for whatever reason the second summon will be bugged. There will be no spawns for the 2nd use of the spawn item but the doors will still come down locking you in. A 3rd summon is possible, but buggy

2. Occasionally the encounter seems to bug, and just reset. I believe I've had resets when suppressors were properly handled, and couldn't explain why. I'm not positive, but it seems to be more likely to happen on the third spawn but may also be related to how fast the fires are doused during the first phase of the encounter. It seems like more enemies spawns in general than before on this server, but especially the Hakkari Bloodkeepers that drop the bloods which I suspect could be causing issues.

I don't have a lot of good notes on how this encounter works, since it can be a rather hectic fleeting experience but it does need some investigation. My characters have completed this quest, but if I attempt this encounter again I'll try to record the experience better.

**Expected Behaviour:**

From wowhead comments it sounds like this boss should only be summonable once per instance reset. It might be preferable to being locked in during a 2nd summon with no mobs though the accuracy is questionable.

There should be a delay between Hakkari Bloodkeeper spawns like time or the death of a Nightmare Suppressor. Though, I've seen several bloodkeepers spawn in a very short time sometimes overlapping, but it could be that extra glitchy 3rd summon. I've been wondering if dousing two flames before killing a suppressor is some kind of sequence break that causes the script to reset but haven't had the patience/group to test that.

The wow classic footage shows less adds than I recall during the encounter, but I'll be sure to check back with confirmation of that if I attempt this encounter again. Perhaps with footage, or screenshots.

https://youtu.be/5cDsUsrsBtA

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=8443
Item: https://database.turtle-wow.org/?item=10460
Item: https://database.turtle-wow.org/?item=10465
Quest: https://database.turtle-wow.org/?quest=3528

**Comment by geojak** [comment]
i have also ecountered unusual short times between bloodseekers and othertimes unsuual long times.
we saw much mire bloodseekers than suppresors too.

**Comment by geojak** [comment]
at one time we had like 2 boodsekr and 1 suprresor spawn like i a spawn of second at once which causedd us to wipe

**Comment by tubtubs** [comment]
I found an opportunity to re-try this boss, someone in my group had the summon item. This is the first attempt/summon, no re-summon required. I focused Suppressors on spawn, as is the expected tactics. Worked out ok in the end, but spawns may be bugged or increased greatly.

https://youtu.be/pP9OGeYj1AM

At 2:55 in the Turtle WoW fight I recorded there are 3 bloodkeepers up at once, before the 2nd fire is lit. This spawn seems duplicated, or something. At least 1 supressor is down but it took us about 30 seconds to do so. In the wow classic source I think this group takes a similiar amount of time to kill the supressor, but we may have been extra behind since our DPS was a little AFK and not as spell cleave heavy perhaps.

I don't know exactly what this fight is supposed to look like but there are a few things that stick out here. The "wow classic" version of this fight I linked seems much much slower paced. Even if DPS were increased greatly, in the footage from wow classic it seems to take at least an extra 2 minutes (or 4ish minutes from encounter start) for 4 bloodkeepers to spawn, where as 4 bloodkeeper spawns happens within just over 2 minutes from starting the encounter on Turtle WoW. 

In fact, it seems like there are 6 blood keepers spawned in about 4 minutes on Turtle WoW. I don't know if this fight is supposed to be timer based or spawn sequentially (blood keeper -> suppressor -> blood keeper -> ...). In any event, there's way more spawns.

I've recorded some time stamps while analyzing the footage to help review my observations.

Turtle WoW time stamps:
~45s start
54s first one spawns
1 31 first one goes down,
1 50 supressor spawns
@255 3 bloodkeepers are up, one has died. This is enough bloods up to finish the fight right now.
-Didnt look around enough to keep track of exactly when they spawned :(
322ish 2nd one dies, no sight of new supressors	
333 3rd one dies
347 4th one is almost dead, supressor spawns
359 fifth blood keeper spawns
430ish 6th blood keeper spawns
438ish 3rd supressor spawns
Remarks:
2ish minutes in 4 blood keepers have spawned


WoW classic time stamps:
216 start
223 first blood keeper spawns
2 47 first blood keeper is dead
329 supressor spawns
356 supressor dies
405ish 2nd blood keeper spawns
432 2nd blood keeper dies
4 55 supressor spawns
529 supressor dies
553 3rd blood dragon dies
612 4th  blood dragon dies
623 next supressor spawns
717 5th  blood dragon spawns 
742ish blood dragon dies

Remarks:
About 4 minutes for 4th dragon spawn
